controller/src/bin/mock-sp.rs

```diff
@@ -0,0 +1,55 @@
+//! Mock SP server that dummies up basic QSFP memory map.
+
+use std::net::Ipv6Addr;
+use std::net::SocketAddrV6;
+use tokio::net::UdpSocket;
+use transceiver_messages::message::HostRequest;
+use transceiver_messages::message::Message;
+use transceiver_messages::message::MessageBody;
+use transceiver_messages::message::SpResponse;
+use transceiver_messages::MAX_PAYLOAD_SIZE;
+use transceiver_messages::PORT;
+
+const PEER: Ipv6Addr = Ipv6Addr::new(0xff02, 0, 0, 0, 0, 0, 0x1de, 2);
```

Comment:
Hmmm, I'd like to use `Ipv6Addr::from(transceiver_messages::ADDR)`, but that's not `const`.

It might be worth it to de-`const` this (moving it into `main`), so that we don't have to specify the IP address in two places?

Reply:
Sure, sounds fine, change incoming.

Reply:
Done in b285b3d